pkg/application/application.go

```diff
@@ -36,6 +36,21 @@ func ListInProject(client *occlient.Client) ([]string, error) {
 	return appNames, nil
 }
 
+// Exists check whether given app exist or not
+func Exists(app string, client *occlient.Client) (bool, error) {
+
+	appList, err := List(client)
```

Comment:
can we use `Get` instead of going through listing all the apps. Something like this https://github.com/openshift/odo/pull/1719/files#diff-6025e9b620e9321c03c11bf29e5c4dbdR474 but for apps?

Reply:
`App` is odo specific object